doc/api/readline.md

```diff
@@ -628,6 +628,15 @@ the best compatibility if it defines an `output.columns` property and emits
 a `'resize'` event on the `output` if or when the columns ever change
 ([`process.stdout`][] does this automatically when it is a TTY).
 
+When creating a `readline.Interface` using `stdin` as input stream, the program
+will notterminate until it receives `EOF` (<kbd>Ctrl</kbd>+<kbd>D</kbd>).
```

Comment:
Interestingly, it seems the manipulation to get `EOF` is different on Windows: https://www.devdungeon.com/content/windows-eof-ctrl-d-equivalent

> <kbd>CTRL</kbd>-<kbd>Z</kbd> followed by <kbd>return</kdb>

Maybe worth adding the info in the parentheses.